src/common/dao/base.go

```diff
@@ -87,14 +87,13 @@ func InitDatabase(database *models.Database) error {
 
 func getDatabase(database *models.Database) (db Database, err error) {
 	switch database.Type {
-	case "", "mysql":
-		db = NewMySQL(database.MySQL.Host,
-			strconv.Itoa(database.MySQL.Port),
-			database.MySQL.Username,
-			database.MySQL.Password,
-			database.MySQL.Database)
-	case "sqlite":
-		db = NewSQLite(database.SQLite.File)
```

Comment:
Is this "sqlite" section removed by mistake?

Reply:
Ok, seem we will not support sqlite

Reply:
yes, will not support sqlite and maria.